components/atoms/ActionButton.js

```diff
@@ -0,0 +1,140 @@
+import PropTypes from "prop-types";
+import Link from "next/link";
+import { useEffect } from "react";
+
+/**
+ * Button component
+ */
+export function ActionButton(props) {
+  //Styling for buttons and links
+  const basicStyle =
```

Comment:
Okay, so we are keeping the action button now? What happens to our other button?

Reply:
I originally used the ActionButton component from Alpha site, wasn't working properly so I created a button component which we decided to get rid of. I'd say we stick with the ActionButton going forward.